org.eclipse.jdt.ls.debug/src/org/eclipse/jdt/ls/debug/adapter/resource/disposer/ResumeThreadResourceDisposer.java

```diff
@@ -0,0 +1,47 @@
+package org.eclipse.jdt.ls.debug.adapter.resource.disposer;
```

Comment:
ResumeThreadResourceDisposer.java looks like same as OnDemandThreadResourceDisposer.java. 
Don't understand why not to merge them.

Reply:
the dispose action for resume request handler and stepinto are different